src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -46,11 +46,15 @@ public CartController(SampleService sampleService, ProjectService projectService
 	 * Get the dedicated page for the Cart
 	 *
 	 * @param model {@link Model}
+	 * @param automatedProject The ID of the automated project to add a pipeline to (optional)
 	 * @return {@link String} path to the cart page template
 	 */
-	@RequestMapping(value = {"", "/*"}, produces = MediaType.TEXT_HTML_VALUE)
-	public String getCartPage(Model model) {
+	@RequestMapping(value = { "", "/*" }, produces = MediaType.TEXT_HTML_VALUE)
+	public String getCartPage(Model model,
+			@RequestParam(required = false, name = "automatedProject")
```

Comment:
Since you updating this anyways can you just add a code comment here that says the contents are loaded using react via `CartTools.jsx`

Reply:
Note this is being done instead in https://github.com/phac-nml/irida/pull/350